viewing_party/main.py

```diff
@@ -0,0 +1,131 @@
+# WAVE One
+def create_movie(movie_title, genre, rating):
+    movie = {}
+    movie['title'] = movie_title
+    movie['genre'] = genre
+    movie['rating'] = rating
+
+    for values in movie.values():
+        if values == None:
+            return None
+
+    return movie
```

Comment:
this works just fine! but we are creating space in memory with a variable `movie` that might no even be valid yet! so let's think about checking validation before making variables and running code that MIGHT not be needed

```suggestion
    if movie_title and genre and rating: # does these values "exist"/are they truthy?
        movie = {"title": movie_title, "genre": genre, "rating": rating}
        return movie
    else:
        return None
```